core/process/query.php

```diff
@@ -3,4 +3,5 @@
 $variables = realpath(dirname(__FILE__)) . '/../json/variables.json';
 $config = json_decode(file_get_contents($variables));
 
-include_once('queries/' . ($config->system->db_type ?: 'rocketmap') . '.php');
\ No newline at end of file
+include_once('queries/' . ($config->system->db_type ?: 'rocketmap') . '.php');
+include_once('queries/' . ($config->system->db_type ?: 'monocle') . '.php');
```

Comment:
remove this line
If db_type is set to monocle via variables.json it will use that. 'rocketmap' is only a fallback.